packages/xo-web/src/common/intl/messages.js

```diff
@@ -2426,11 +2426,11 @@ const messages = {
   templatesLabel: 'Templates',
   recipesLabel: 'Recipes',
   network: 'Network',
-  recipeMasterNameLabel: 'Master node name',
+  recipeControlPlaneNameLabel: 'Control plane node name',
```

Comment:
Change also needed in [locales/it.js](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/intl/locales/it.js#L5666)